website/docs/examples/camera_aruco_detector/README.md

```diff
@@ -0,0 +1,241 @@
+cd farm-ng-amiga/py/examples/camera_aruco_detector
+```
+
+```bash
+pip3 install -r requirements.txt
+```
+
+This is will install the following dependencies:
+    - `farm-ng-amiga` - the farm-ng python client library
+    - `opencv-python` - the OpenCV python library
+    - `opencv-contrib-python` - the OpenCV python library with extra modules (needed for Aruco detection)
+
+### 3. The Aruco detector
+
+In order to detect the Aruco markers, we need to create an Aruco detector object and
+configure it with the desired parameters. To do this, we'll create an auxiliary class
+`ArucoDetector` that will hold the detector object and the configuration parameters.
+
+In this example, we'll use the `DICT_6X6_250` Aruco dictionary type and a marker size of 0.1 meters
+to detect the Aruco markers following the [**OpenCV Aruco tutorial**](https://docs.opencv.org/master/d5/dae/tutorial_aruco_detection.html).
+
+```python
+class ArucoDetector:
+    """A class for detecting ArUco markers in an image frame."""
+
+    def __init__(self, aruco_dict_type: str, marker_size: float) -> None:
+        """Initialize the ArUco detector.
+
+        Args:
+            aruco_dict_type (str): The ArUco dictionary type.
+            marker_size (float): The size of the ArUco marker in meters.
+        """
+        self._detector = self._create_detector(aruco_dict_type)
+        self._marker_size = marker_size
+
+    def _create_detector(self, aruco_dict_type: str) -> cv2.aruco.ArucoDetector:
+        """Create an ArUco detector.
+
+        Args:
+            aruco_dict_type (str): The ArUco dictionary type.
+
+        Returns:
+            cv2.aruco.ArucoDetector: The ArUco detector.
+        """
+        aruco_params = cv2.aruco.DetectorParameters()
+
+        # See all the available ArUco dictionary types here:
+        # https://docs.opencv.org/4.x/de/d67/group__objdetect__aruco.html#ga4e13135a118f497c6172311d601ce00d
+        aruco_dict = cv2.aruco.getPredefinedDictionary(getattr(cv2.aruco, aruco_dict_type))
+        return cv2.aruco.ArucoDetector(aruco_dict, aruco_params)
+```
+
+We will also create a couple of internal utilities to create the reference 3d points needed by the
+pose estimation solver.
+
+```python
+    def _create_object_points(self, marker_size: float) -> np.ndarray:
+        """Create the object points for the ArUco markers.
+
+        Args:
+            marker_size (float): The size of the ArUco marker in meters.
+
+        Returns:
+            np.ndarray: The object points for the ArUco markers.
+        """
+        size_half: float = marker_size / 2.0
+        return np.array(
+            [
+                [-size_half, -size_half, 0],
+                [size_half, -size_half, 0],
+                [size_half, size_half, 0],
+                [-size_half, size_half, 0],
+            ],
+            dtype=np.float32,
+        )
+```
+
+As described in the Camera to Pointcloud tutorial, we need to create a camera intrinsics matrix
```

Comment:
Add link to pointcloud tutorial. Or if recommended to go through first, add link at the top with camera client and camera calib